eng/scripts/Detect-ApiChanges.ps1

```diff
@@ -0,0 +1,129 @@
+[CmdletBinding()]
```

Comment:
minor but you renamed this file from what is currently in common. In Common this is called Detect-Api-Changes.ps1, not sure if that is intentional or not but wanted to call it out.

Reply:
It was intentional change but not really needed. I will update the name.

Reply:
Removed this file. I am changing common script